docs/source/nodes_and_pipelines/namespaces.md

```diff
@@ -0,0 +1,336 @@
+# Reuse pipelines with namespaces
+
+## How to reuse your pipelines
+
+If you want to create a new pipeline that performs similar tasks with different inputs/outputs/parameters as your `existing_pipeline`, you can use the same `pipeline()` creation function as described in [How to structure your pipeline creation](modular_pipelines.md#how-to-structure-your-pipeline-creation). This function allows you to overwrite inputs, outputs, and parameters. Your new pipeline creation code should look like this:
+
+```python
+def create_new_pipeline(**kwargs) -> Pipeline:
+    return pipeline(
+    [existing_pipeline], # Name of the existing pipeline
+    inputs = {"old_input_df_name" : "new_input_df_name"},  # Mapping old input to new input
+    outputs = {"old_output_df_name" : "new_output_df_name"},  # Mapping old output to new output
+    parameters = {"params: model_options": "params: new_model_options"},  # Updating parameters
+    )
+```
+
+This means you can create multiple pipelines based on the `existing_pipeline` pipeline to test different approaches with various input datasets and model training parameters. For example, for the `data_science` pipeline from our [Spaceflights tutorial](../tutorial/add_another_pipeline.md#data-science-pipeline), you can restructure the `src/project_name/pipelines/data_science/pipeline.py` file by separating the `data_science` pipeline creation code into a separate `base_data_science` pipeline object, then reusing it inside the `create_pipeline()` function:
+
+```python
+#src/project_name/pipelines/data_science/pipeline.py
+
+from kedro.pipeline import Pipeline, node, pipeline
+from .nodes import evaluate_model, split_data, train_model
+
+base_data_science = pipeline(
+        [
+            node(
+                func=split_data,
+                inputs=["model_input_table", "params:model_options"],
+                outputs=["X_train", "X_test", "y_train", "y_test"],
+                name="split_data_node",
```

Comment:
And similarly in other nodes, because in all the places this `name` is used, it is clear that it's a `node`.
```suggestion
                name="split_data",
```

Reply:
I would prefer to leave this as it is because it's currently how it's written in our starters, and I don't want to confuse users with different naming since our example is based on that starter.